genai-perf/genai_perf/config/generate/perf_analyzer_config.py

```diff
@@ -359,6 +360,30 @@ def _add_extra_args(self, extra_args: Optional[List[str]]) -> List[str]:
 
         return args
 
+    def _calculate_request_count(self, config: ConfigCommand) -> int:
+        REQUEST_COUNT_CONCURRENCY_MULTIPLIER = 2
+
+        config_request_count = config.perf_analyzer.measurement.num
+        concurrency = self._get_concurrency(config)
+
+        request_count = max(
```

Comment:
I think this could lead to unexpected behavior. 
If a user sets a request count less than concurrency * 2, this will not respect the user's input as I currently understand it.

Reply:
fixed now. Code will respect a user provided request count.

Reply:
That's a very good point. I've changed the logic to now respect if the user has set a value for this field. However, if we are inferring a value (blank in the config, or not set on the CLI) then we will use the max(10, 2 * concurrency)